lib/active_fedora/solr_query_builder.rb

```diff
@@ -35,6 +35,12 @@ def self.construct_query_for_rel(field_pairs, join_with = 'AND')
       clauses.empty? ? "id:NEVER_USE_THIS_ID" : clauses.join(" #{join_with} ".freeze)
     end
 
+    # Adds esaping for spaces which are not handled by RSolr.solr_escape
+    # See rsolr/rsolr#101
+    def self.solr_escape terms
+      RSolr.solr_escape(terms).gsub(/\s+/,"\\ ")
+    end
```

Comment:
@cam156 I don't think so. RSolr was escaping spaces before, but solr doesn't *need* spaces escaped. The escaped spaces just gives the query a different meaning.